paddle/fluid/framework/details/all_reduce_op_handle.cc

```diff
@@ -45,6 +46,7 @@ AllReduceOpHandle::AllReduceOpHandle(ir::Node *node,
 #endif
 
 void AllReduceOpHandle::RunImpl() {
+  platform::RecordEvent r("all_reduce", nullptr);
```

Comment:
Is every op need to add this line? I notice that prefetch op is not in the profiler report

Reply:
For those op_handle with no OpDesc in it, you need to add one in OpHandle.